### PR TITLE
WIP condafication of Visceral-Project segmentation evaluation tool (ubunt…

### PR DESCRIPTION
**Pull request by ThomasWollmann**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

…u only)

### CODE REVIEW

File: recipes/visceral-evaluatesegmentation/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package:
+  name: visceral-evaluatesegmentation
+  version: "2015.07.02"
+
+source:
+  fn: EvaluateSegmentation-2015.07.02-Ubuntu.zip # [linux]
+  url: https://github.com/Visceral-Project/EvaluateSegmentation/raw/master/builds/Ubuntu/EvaluateSegmentation-2015.07.02-Ubuntu.zip # [linux]
+
+build:
+  number: 1
```
Comment (bgruening) [inline recipes/visceral-evaluatesegmentation/meta.yaml]: can you start with 0

File: recipes/visceral-evaluatesegmentation/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package:
+  name: visceral-evaluatesegmentation
+  version: "2015.07.02"
+
+source:
+  fn: EvaluateSegmentation-2015.07.02-Ubuntu.zip # [linux]
+  url: https://github.com/Visceral-Project/EvaluateSegmentation/raw/master/builds/Ubuntu/EvaluateSegmentation-2015.07.02-Ubuntu.zip # [linux]
```
Comment (bgruening) [inline recipes/visceral-evaluatesegmentation/meta.yaml]: Can you compile this one and not using the rebuild-binary? I'm pretty sure this binary will not work on HPC environments.

File: recipes/visceral-evaluatesegmentation/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+build:
+  number: 1
+
+#test:
```
Comment (bgruening) [inline recipes/visceral-evaluatesegmentation/meta.yaml]: Test are strictly needed in bioconda, did you had some problems with it?

File: recipes/visceral-evaluatesegmentation/meta.yaml (hunk 2 above)
Comment (ThomasWollmann) [inline recipes/visceral-evaluatesegmentation/meta.yaml]: Had no time yet to build it from scratch. Sadly it is not documented and linking to an unknown ITK can be tricky.

Comment (bgruening) [inline recipes/visceral-evaluatesegmentation/meta.yaml]: Than go for the binary, but it will probably not work on HPC.

File: config.yml [hunk 4]
```diff
@@ -13,4 +13,5 @@ channels:
     - r
     - defaults
     - conda-forge
+    - heimdali
```
Comment (bgruening) [inline config.yml]: This is not allowed. But you can put the needed recipe into bioconda, conda-forge as well.

File: recipes/quake/build.sh [hunk 5]
```diff
@@ -1,4 +1,3 @@
-export CC=${PREFIX}/bin/gcc
 export CXX=${PREFIX}/bin/g++
```
Comment (bgruening) [inline recipes/quake/build.sh]: why is this commit in your branch?

Comment (ThomasWollmann) [inline recipes/quake/build.sh]: I don't know ;)

File: config.yml (hunk 4 above)
Comment (ThomasWollmann) [inline config.yml]: :/

**Comment by ThomasWollmann** [comment]
Will wait for ITK package... https://github.com/bioconda/bioconda-recipes/pull/3359

**Comment by ThomasWollmann** [comment]
Now depends on https://github.com/bioconda/bioconda-recipes/pull/3439

**Comment by ThomasWollmann** [comment]
Continues at https://github.com/bioconda/bioconda-recipes/pull/3468

**Pull request closed without merging**